.github/workflows/mig.yml

```diff
@@ -0,0 +1,82 @@
+name: mig-test
+
+on:
+  pull_request:
+    branches:
+      - main
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+      - name: setup go environment
+        uses: actions/setup-go@v1
+        with:
+          go-version: '1.15.4'
+      - name: download dependencies
+        run: make bootstrap
+      - name: run unit tests
+        run: make test
+      - name: build binary
+        run: make build-linux
+      - name: run acceptance tests
+        run: sudo pip install virtualenv && make acceptance
+      - name: Prepare
+        id: prepare
+        run: |
+          DOCKER_IMAGE=chartmuseum/chartmuseum
```

Comment:
I think the issue is this probably needs to be `ghcr.io/helm/chartmuseum`

Reply:
okay i see